source/test/index.twig

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8"/>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1"/>
+  <meta name="description" content="Testing CSSgram filters"/>
+  <meta name="author" content="Miles Croxford"/>
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+  <link rel="stylesheet" href="../source/css/cssgram.min.css">
+  <title>CSSgram test sheet</title>
+  <link rel="stylesheet" href="css/test-site.css">
+</head>
+<body>
+
+  {% set filters = [
+  {name: 'aden',       is_done: true}, 
+  {name: 'amaro',      is_done: false}, 
+  {name: 'brannan',    is_done: false},
+  {name: 'crema',      is_done: false}, 
+  {name: 'earlybird',  is_done: true}, 
+  {name: 'hefe',       is_done: false}, 
+  {name: 'hudson',     is_done: true}, 
+  {name: 'inkwell',    is_done: true}, 
+  {name: 'juno',       is_done: false}, 
+  {name: 'lark',       is_done: false}, 
+  {name: 'lofi',       is_done: true}, 
+  {name: 'ludwig',     is_done: false}, 
+  {name: 'mayfair',    is_done: true}, 
+  {name: 'nashville',  is_done: false},
+  {name: 'perpetua',   is_done: true}, 
+  {name: 'reyes',      is_done: true}, 
+  {name: 'rise',       is_done: false}, 
+  {name: 'sierra',     is_done: false}, 
+  {name: 'slumber',    is_done: false}, 
+  {name: 'valencia',   is_done: false}, 
+  {name: 'willow',     is_done: false}, 
+  {name: 'xpro2',      is_done: true}, 
+  ] %}
+
+  {% set images = ['atx', 'bike', 'cacti', 'lakegeneva', 'tahoe'] %}
+
+  <header>
+    <img class="logo" src="../site/img/cssgram-logo.svg" alt="CSSgram logo"> <h1 class="blurb"> – Test sheet </h1>
+  </header>
+
+  {% for f in filters %}
+  <section id="{{f.name}}">
+    <h2 class="filter-header"><a href="#{{f.name}}">{{f.name|title}}</a> <span class="done  done--{{f.is_done ? 'yep' : 'nope'}}">{{f.is_done ? '✓' : '✗'}}</span></h2>
+    <div class="headers">
+      <h3>Instagram</h3>
+      <h3>CSSgram</h3>
+    </div>
+    {% for i in images %}
+    <section class="test-images">
+
+     {# ig #}
+     <div>
+      <div>
+        <figure>
+          <img src="ref/{{i}}-{{f.name}}.jpg" alt="Instagram version of {{f.name}}">
+        </figure>
+      </div>
+    </div>
+
+    {# cssgram #}
+    <div>
```

Comment:
Get rid of all of these divs, please! This is a bit cleaner:

```
     {# ig #}
        <figure>
          <img src="ref/{{i}}-{{f.name}}.jpg" alt="Instagram version of {{f.name}}">
        </figure>

    {# cssgram #}
      <figure class="{{f.name}}">
          <img src="../site/img/{{i}}.jpg" alt="CSSgram version of {{f.name}}"></div>
        </figure>
```

Just the `<figure>` tags are all you need